src/Tribe/Cron.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+class Tribe__Cron {
+
+	/**
+	 * Filters the cron schedules to add intervals we need.
+	 *
+	 * @param array $schedules
+	 *
+	 * @return array
+	 */
+	public function filter_cron_schedules( array $schedules ) {
+		$schedules[ 'thirty_seconds' ] = array(
+			'interval' => 30,
```

Comment:
Older versions will not allow for that, but newer versions will. 
In a service project we have a 20" cron.